google-cloud-storage-control/src/test/java/com/google/storage/control/v2/MockStorageControlImpl.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.storage.control.v2;
+
+import com.google.api.core.BetaApi;
+import com.google.longrunning.Operation;
+import com.google.protobuf.AbstractMessage;
+import com.google.protobuf.Empty;
+import com.google.storage.control.v2.StorageControlGrpc.StorageControlImplBase;
+import io.grpc.stub.StreamObserver;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+import javax.annotation.Generated;
+
+@BetaApi
+@Generated("by gapic-generator-java")
+public class MockStorageControlImpl extends StorageControlImplBase {
```

Comment:
@suztomo this mock helper class relies on gapic- artifact; what do you recommend we do in this case?

Reply:
Good catch. The usage was overlooked. Memo: this is the code. https://github.com/googleapis/googleapis-gen/blob/114ca4d376e84533150088df8ac86e098241e966/google/storage/control/v2/google-cloud-storage-control-v2-java/grpc-google-cloud-storage-control-v2-java/src/main/java/com/google/storage/control/v2/StorageControlGrpc.java#L4

Assuming this MockStorageControlImpl is needed, I think it's reasonable to bring the "grpc-" artifact in OwlBot.yaml. I'll work on this tomorrow.